fmn/api/main.py

```diff
@@ -72,20 +72,25 @@ async def read_root(
     return result
 
 
-@app.get("/users/")
+@app.get("/users/", response_model=list[api_models.User])
 async def get_users(
     db_session: AsyncSession = Depends(gen_db_session),
 ):  # pragma: no cover todo
     result = await db_session.execute(select(User))
-    return {"users": result.scalars().all()}
+    return result.scalars().all()
 
 
-@app.get("/user/{username}/")
-async def get_user(
-    username, db_session: AsyncSession = Depends(gen_db_session)
+@app.get("/fasjsonusers/")
+def get_fasjsonusers(
+    fasjson_client: FasjsonClient = Depends(get_fasjson_client),
 ):  # pragma: no cover todo
+    return [u["username"] for u in fasjson_client.list_all_entities("users")]
```

Comment:
This is going to take way too long. I think it should forward a search for the username to fasjson.